pkg/deploy/controller/imagechange/controller.go

```diff
@@ -111,10 +111,10 @@ func (c *ImageChangeController) Handle(stream *imageapi.ImageStream) error {
 	}
 
 	if anyFailed {
-		return fatalError(fmt.Sprintf("couldn't update some deployment configs for trigger on image stream %q", imageapi.LabelForStream(stream)))
```

Comment:
I thought returning a fatalError here just kept this error from causing an immediate requeue... I'm not seeing how it was preventing subsequent image change events from triggering attempts. Is the title accurate, or are we always wanting to retry immediately now?